.github/workflows/composite/comment-on-pr/action.yml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022 The Magma Authors.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Comment on PR
```

Comment:
Why did you outsource this into an action?

Reply:
We have several other actions, which do the same stuff with a very very complex and bloated workflow. Thus outsourcing this will make it easy to simplify those other workflows as well and drastically reduce the complexity.

Reply:
Which other actions do you have in mind? Can you provide a link?

Reply:
Amended further simplification to this PR, as I did not use the full potential before.
For starters the DCO check could be greatly simplified. See last commit of https://github.com/magma/magma/pull/14475.

Reply:
- Python Format
- Docs Lint

are further candidates.